### PR TITLE
Nifireg 279

### PR DESCRIPTION
**Pull request by scottyaslan**
https://issues.apache.org/jira/browse/NIFIREG-279

### CODE REVIEW

**Comment by mcgilman** [comment]
Will review...

File: nifi-registry-core/nifi-registry-web-ui/src/main/webapp/theming/nf-registry.scss [hunk 1]
```diff
@@ -36,8 +35,8 @@ $mat-font-url: './assets/fonts/';
 
 $primaryColor: $rose1; //$green2
 $primaryColorHover: $rose2; //$green3
-$accentColor: $blue7; //$orange1
-$accentColorHover: $grey4; //$orange2
+$accentColor: $blue-grey1; //$orange1
```
[the remark this answers is not in the document]
Comment (rfellows) [reply nifi-registry-core/nifi-registry-web-ui/src/main/webapp/theming/nf-registry.scss]: Not sure it has to do with using the latest FDS or not, but it is not clear at first glance which tab is selected
Screen Shot 2019-08-01 at 5 27 04 PM

**Comment by mcgilman** [comment]
@scottyaslan Thanks for the PR! I'm also seeing many differences from our previous FDS dependency. The fonts, spacing, coloring, etc are all a little different. We should get @moranr to also take a peek.

File: nifi-registry-core/nifi-registry-web-ui/src/main/webapp/theming/nf-registry.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (scottyaslan) [reply nifi-registry-core/nifi-registry-web-ui/src/main/webapp/theming/nf-registry.scss]: These concerns have been addressed in the latest commit.

**Comment by scottyaslan** [comment]
@mcgilman @rfellows this PR is ready for review.

**Comment by rfellows** [comment]
reviewing...

**Comment by mcgilman** [comment]
Thanks @scottyaslan! This has been merged to master.

**Pull request closed without merging**